lua/neotest-rspec/utils.lua

```diff
@@ -0,0 +1,62 @@
+local logger = require("neotest.logging")
+
+local M = {};
+
+---@param position_id string
+---@return string
+M.form_treesitter_id = function(position_id)
```

Comment:
I'd like to refactor this method a bit but don't want to make this PR any more complicated. Will proposed something in a subsequent PR.